test/libsolidity/ASTJSON/address_payable.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+    mapping(address => address payable) public m;
+    function f(address payable arg) public returns (address payable r) {
+        address payable a = m[arg];
+        r = arg;
+		address c = address(this);
+        m[c] = address(0);
+    }
+}
+
```

Comment:
We can use a simpler test case in the end in order not to blow up the size of the test expectations, but I wanted to have a look at all ``address`` constructs in the JSON AST.